examples/ffmpeg-transcode.cpp

```diff
@@ -204,8 +204,6 @@ static int decode_audio(struct audio_buffer *audio_buf, s16 **data, int *size)
     const size_t errbuffsize = 1024;
     char errbuff[errbuffsize];
 
-    av_register_all(); // from avformat. Still a must-have call for ffmpeg v3! (can be skipped for later versions)
```

Comment:
naive and bad modif: breaks the ffmpeg v3 build.
Proper modif should have been:
#ifdef FFMPEG_VERSION < 5
 av_register_all();
#endif

Reply:
If you want this to happen just do a PR with:

```
#ifdef FFMPEG_VERSION < 4
av_register_all();
#endif
```

In anyway this is an example for the current state and meant to be simple.

I find the tone quite harsh and not inviting for help.
The caveat for ffmpeg 3.0 was clearly mentioned in the description though.
The call is deprecated since ffmpeg 4.0 in 2018.